src/artifacts.ts

```diff
@@ -5,12 +5,19 @@ import axios, {AxiosRequestConfig, AxiosResponse} from 'axios'
 import axiosRetry from 'axios-retry'
 import fs from 'fs'
 
+export async function workflowName(
+  workflow?: string | undefined
+): Promise<string> {
+  if (workflow === undefined) {
+    workflow = github.context.workflow
+  }
+  return workflow.replace(/\W/g, '-').replace(/^-/, '')
```

Comment:
why do we need the replace ?

Reply:
if a workflow is named like the following: `workflow name` we are replacing ` ` characters (or `\W` in regex) to `-` so that the workflow name becomes, `workflow name`

Note this is not a new code, this is the code used to generate the workflow name in the setup phase and I just made the get artifacts method use the same filter/replace/function

Reply:
We should create a function that do this job, with a clear name